portfolio/src/main/webapp/script.js

```diff
@@ -16,8 +16,12 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
+  const frase1 = 'You can tell me Brandocean';
+  const frase2 = 'I like watching anime';
+  const frase3 = 'I have 3 dogs';
+  const frase4 = 'I study at Tec de Monterrey in Cuernavaca, Morelos';
   const greetings =
-      ['You can tell me Brandocean', 'I like watching anime', 'I have 3 dogs', 'I study at Tec de Monterrey in Cuernavaca, Morelos'];
+      [frase1, frase2, frase3, frase4];
```

Comment:
FYI since these constants are not used anywhere else, you could've kept what you had and just separated each element in its own line like in [this example](https://google.github.io/styleguide/jsguide.html#formatting-array-literals)